doc/sphinx/changes.rst

```diff
@@ -1197,6 +1197,66 @@ Miscellaneous
   (`#19842 <https://github.com/rocq-prover/rocq/pull/19842>`_,
   by Gaëtan Gilbert).
 
+Changes in 9.0.1
+~~~~~~~~~~~~~~~~
+
+.. contents::
+   :local:
+
+Kernel
+^^^^^^
+
+- **Fixed:**
+  Guard checking forgot to check non principal arguments of a fixpoint
+  for unguarded uses of the fixpoint leading to an inconsistency
+  (`#20415 <https://github.com/coq/coq/pull/20415>`_,
```

Comment:
```suggestion
  (`#20415 <https://github.com/coq/coq/pull/20415>`__,
```
or match the other instance of this link text
```suggestion
  (`#20415 <https://github.com/rocq-prover/rocq/pull/20415>`_,
```

Reply:
see https://github.com/sphinx-doc/sphinx/issues/3921#issuecomment-315581557 for more info